### PR TITLE
Replace BC logo with hi-res transparent version

### PR DESCRIPTION
**Pull request by TrevorOctober**

### CODE REVIEW

**Comment by TrevorOctober** [comment]
@miltontony Ready for review

**Comment by TrevorOctober** [comment]
@miltontony @saeedmb8 Please review

File: unicorecmsmama/templates/base.pt [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
     <div id="header">
         <a href="/">
-          <img src="/static/img/logo-bc.png?v=2" border="0" />
+          <img src="/static/img/baby_h_rgb_5c.png?v=2" border="0" />
```
Comment (hodgestar) [inline unicorecmsmama/templates/base.pt]: Should we keep the name of the file `logo-bc.png` and just update `v=2` to `v=3`?

**Comment by TrevorOctober** [comment]
@hodgestar @miltontony Ready for re-review

**Comment by hodgestar** [comment]
You should remove `baby_h_rgb_5c.png` now, but otherwise looks perfect.

**Comment by miltontony** [comment]
once you've removed the duplicate file :+1:

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**